usi/src/main/proto/usi.proto

```diff
@@ -63,8 +63,8 @@ message SwitchInfo {
   // IP address of external switch.
   string ip_addr = 1;
 
-  // Telnet Port
-  int32 telnet_port = 2;
+  // telnet port for sending switch commands.
```

Comment:
drop the "telnet" -- just cmd_port -- no need to specify the protocol here.

Reply:
Puneet and I both think cmd_port is too ambiguous. cmd_port can be construed as the openflow port or the switch control port. Too many things are named ports in the current configuration where some ports are for physical ports and others are for tcp ports.